### PR TITLE
Survey styling

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds styling for surveys on the special projects template

screencapture-citylimits-test-series-the-class-of-2013-2019-11-11-14_20_55

## Why

For #72

## Testing/Questions

Features that this PR affects:

- Gravity form widgets placed on special projects where the form contains the class `special-projects-survey`

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] Does it look ok?
- [ ] What other elements should we style?
- [ ] Should the textarea be the same width as the normal input? If so, the right side of the form will look very empty and bare. Is that ok?

Steps to test this PR:

1. Modify an existing GF survey (I used ID #6 Job Survey)
2. Under "Form Settings" add `special-projects-survey` to the "CSS Class Name" field
3. Add in different types of form fields
4. Add the Gravity Form widget to a special projects widget area
   - Title: empty
   - Select a form: The one you edited above
   - Display form title: yes
   - Display description: yes

### CODE REVIEW

File: wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-featured-content-widget.php [hunk 1]
```diff
@@ -151,6 +151,7 @@ function widget( $args, $instance ) {
 			// close the ul
 			echo '</ul>';
 
+			echo '</div>';
```
Comment (benlk) [inline wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-featured-content-widget.php]: Where does this come from?

Comment (joshdarby) [reply wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-featured-content-widget.php]: Good question. Removed in https://github.com/INN/umbrella-citylimits/pull/106/commits/92a750e1acf91d89d8727d25121f2a2cfa36dd61

**Comment by benlk** [comment]
Screen Shot 2019-11-11 at 17 52 58

On Chrome and Firefox, it looks like the height on `input type="text"` is 0, plus padding. This is a Largo bug: https://github.com/INN/largo/issues/1830 but we should have a shim here.

If you're not seeing that, here's my plugin versions:

```
+--------------------------------------+----------+------------------------------+---------+
| name                                 | status   | update                       | version |
+--------------------------------------+----------+------------------------------+---------+
| google-tag-manager                   | active   | none                         | 1.0.2   |
| gravityforms                         | active   | available                    | 2.4.14  |
| gravity-forms-custom-post-types      | active   | none                         | 3.1.15  |
| gravityformspaypal                   | active   | none                         | 3.2     |
| gravityformswebhooks                 | active   | none                         | 1.2     |
```

**Comment by benlk** [comment]
The error conditions should probably be styled:

Screen Shot 2019-11-11 at 17 59 21

**Comment by joshdarby** [comment]
@benlk https://github.com/INN/umbrella-citylimits/pull/106/commits/5ab091d57822cf9b5bcdb890b645e3fc9811a537 updates the minimum height for the inputs to `40px;`

Also styles the errors to look like this. I'm open to any other suggestions on how to style them
Screen Shot 2019-11-12 at 5 23 24 PM

**Comment by benlk** [comment]
Odd thing going on in this screenshot: the City Limits special projects featured content widget has eaten all later widgets on the page:

Screenshot_2019-11-13 Election Watch 2018 City Limits

There's a fix for this in https://github.com/INN/umbrella-citylimits/pull/112/ in the Special Projects Featured Content Widget.

**Pull request merged**